src/all/ahottie/src/eu/kanade/tachiyomi/extension/all/ahottie/AHottie.kt

```diff
@@ -0,0 +1,110 @@
+package eu.kanade.tachiyomi.extension.all.ahottie
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.Request
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class AHottie() : ParsedHttpSource() {
+    override val baseUrl = "https://ahottie.net"
+    override val lang = "all"
+    override val name = "AHottie"
+    override val supportsLatest = false
+
+    // Popular
+    override fun popularMangaFromElement(element: Element) = SManga.create().apply {
+        thumbnail_url = element.select(".relative img").attr("src")
+        genre = element.select(".flex a").joinToString(", ") {
+            it.text()
+        }
+        title = element.select("h2").text()
+        setUrlWithoutDomain(element.select("a").attr("href"))
+    }
+
+    override fun popularMangaNextPageSelector() = "a[rel=next]"
+    override fun popularMangaRequest(page: Int): Request {
+        return GET("$baseUrl/?page=$page")
+    }
+
+    override fun popularMangaSelector() = "#main > div > div"
+
+    // Search
+
+    override fun searchMangaFromElement(element: Element) = popularMangaFromElement(element)
+    override fun searchMangaNextPageSelector() = popularMangaNextPageSelector()
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        return when {
+            query.isNotEmpty() -> GET("$baseUrl/search?kw=$query&page=$page")
+            else -> popularMangaRequest(page)
+        }
+    }
+
+    override fun searchMangaSelector() = popularMangaSelector()
+
+    // Details
+    override fun mangaDetailsParse(document: Document) = SManga.create().apply {
```

Comment:
if there is no other details to fetch, consider putting `initialized = true` in the objects returned by popular/latest/search so an unnecessary call can be avoided